torch/_dynamo/trace_rules.py

```diff
@@ -2765,11 +2765,6 @@
         "torch.nn.grad.conv2d_weight",
         "torch.nn.grad.conv3d_input",
         "torch.nn.grad.conv3d_weight",
-        "torch.nn.modules.activation._is_make_fx_tracing",
-        "torch.nn.modules.utils._list_with_default",
-        "torch.nn.modules.utils._ntuple",
-        "torch.nn.modules.utils._quadruple",
```

Comment:
It is possible that this will fail some test named `test_pair` ... I tried that earlier and saw some failures.